Classes/VideoGameRepository.cs

```diff
@@ -267,5 +267,14 @@ public VideoGameRepository()
             pc.AddGame(satisfactory);
             pc.AddGame(celeste);
         }
+
+        public void PrintGamesByStudio(string studio)
+        {
+            IEnumerable<VideoGame> videogameList = VideoGames.Where(m => m.Studio.Name == studio).ToList();
```

Comment:
There is an easier way to do this.  The Studio class has a VideoGames collection.  You could invoke it with
foreach (var vg in studio.VideoGames)